src/main/java/app/service/BookService.java

```diff
@@ -11,5 +11,7 @@ public interface BookService {
 
     List<BookDto> getAllBooks();
 
-    boolean deleteBook(Long id);
+    void deleteById(Long id);
+
+    void update(Long id, CreatBookRequestDto creatBookRequestDto);
```

Comment:
There is a typo in the parameter name 'creatBookRequestDto'. It should be 'createBookRequestDto'. Correcting this will improve code readability and prevent potential errors.